rfc/005-generate-injection.md

```diff
@@ -50,7 +50,17 @@ To make `contrast generate` idempotent and handle potential edge cases, we will
 
 ### Service Mesh Injection
 
-The service mesh injection will follow a similar approach to the initializer injection outlined, possibly with additional configuration for labels for service discovery/ports. Details of the service mesh injection will be expanded in a future PR update to this RFC.
+The service mesh injection will follow a similar approach to the initializer injection outlined.
+During the generation process, all containers with the `contrast-cc` runtime class will have a service mesh added as a sidecar init container.
+This behavior can be disabled with the command-line flag `--skip-servicemesh` or the per-workload annotation `contrast.edgeless.systems/skip-servicemesh` set to `true`.
+
+The configuration for the Envoy proxy is handled via Kubernetes object annotations.
+The annotations `contrast.edgeless.systems/sm-ingress`, `contrast.edgeless.systems/sm-egress` and `contrast.edgeless.systems/sm-admin-interface-port` will be written into the environment variables `EDG_INGRESS_PROXY_CONFIG`, `EDG_EGRESS_PROXY_CONFIG` and `EDG_ADMIN_PORT` of the injected service mesh container.
```

Comment:
Please spell out `servicemesh` in these annotations, this will make the purpose obvious even for someone unfamiliar with the topic.